pdns/ws-auth.cc

```diff
@@ -2250,6 +2250,28 @@ static void apiServerSearchData(HttpRequest* req, HttpResponse* resp) {
   resp->setBody(doc);
 }
 
+static void apiServerSubZones(HttpRequest* req, HttpResponse* resp) {
+  if(req->method != "GET")
+    throw HttpMethodNotAllowedException();
+
+  DNSName zonename = apiZoneIdToName(req->parameters["id"]);
+  Json::array doc;
+  vector<std::tuple<string, string>> subZoneResult;
+  UeberBackend B;
+  B.getSubZones(zonename.toStringNoDot(), subZoneResult);
+  for(const std::tuple<string, string> &z: subZoneResult)
+  {
+    auto object = Json::object {
+        { "object_type", "subzone" },
+        { "zone_id", std::get<0>(z) },
```

Comment:
is the integer id useful here -- maybe you wanted `apiZoneNameToId` on the zone name instead?

if this endpoint returned the same type as `apiServerZones`, one wouldn't need to introduce a new object_type (and the documentation for it).